bin/check-ses-limit.rb

```diff
@@ -29,20 +30,20 @@
 #
 
 require 'sensu-plugin/check/cli'
-require 'aws/ses'
+require 'sensu-plugins-aws'
+require 'aws-sdk'
 
 class CheckSESLimit < Sensu::Plugin::Check::CLI
+  include Common
   option :aws_access_key,
          short: '-a AWS_ACCESS_KEY',
          long: '--aws-access-key AWS_ACCESS_KEY',
-         description: "AWS Access Key. Either set ENV['AWS_ACCESS_KEY'] or provide it as an option",
-         default: ENV['AWS_ACCESS_KEY']
+         description: 'AWS Access Key ID.'
 
   option :aws_secret_access_key,
          short: '-k AWS_SECRET_KEY',
          long: '--aws-secret-access-key AWS_SECRET_KEY',
-         description: "AWS Secret Access Key. Either set ENV['AWS_SECRET_KEY'] or provide it as an option",
-         default: ENV['AWS_SECRET_KEY']
+         description: 'AWS Secret Access Key.'
```

Comment:
It doesn't break the current environment variable functionality.

The default credential provider checks for credentials in the order that you can see at:
https://github.com/aws/aws-sdk-ruby/blob/master/aws-sdk-core/lib/aws-sdk-core/credential_provider_chain.rb#L20-L31

For environment variables, it checks 3 sets of names:
https://github.com/aws/aws-sdk-ruby/blob/master/aws-sdk-core/lib/aws-sdk-core/credential_provider_chain.rb#L41-L46

I also tested it to make sure.

It is a little bit scary because the documentation only says it supports AWS_ACCESS_KEY_ID & AWS_SECRET_ACCESS_KEY.